src/renderer/components/watch-video-info/watch-video-info.js

```diff
@@ -102,6 +102,10 @@ export default defineComponent({
     videoThumbnail: {
       type: String,
       required: true
+    },
+    inUserPlaylist: {
+      type: Boolean,
+      default: false
```

Comment:
Why default false instead of make it required
The only place using `watch-video-info` is `src/renderer/views/Watch/Watch.vue`

Reply:
```suggestion
      required: true
```